cmd/main.go

```diff
@@ -108,6 +110,8 @@ func setupCommonFlags() {
 		fs.StringVar(&neo4jURI, "neo4j-uri", "neo4j://localhost:7687", "The Neo4j URI.")
 		fs.StringVar(&neo4jUser, "neo4j-user", "neo4j", "The Neo4j user.")
 		fs.StringVar(&neo4jPassword, "neo4j-password", "", "The Neo4j password.")
+		fs.StringVar(&githubApiURI, "github-api-uri","https://api.github.com", "The target GitHub API URI, defaults to https://api.github.com; Change if targeting Github Enterprise.")
```

Comment:
since these are only relevant for the GitHub command they should be subcommand flags (see `githubToken` above)

Reply:
latest commit should fix this - let me know if this is what you were referring to.